common/app/model/Cached.scala

```diff
@@ -21,6 +21,7 @@ object CacheTime {
   object RecentlyUpdated extends CacheTime(60)
   // There is lambda which invalidates the cache on press events, so the facia cache time can be high.
   object Facia extends CacheTime(60, Some(900))
+  object Showcase extends CacheTime(60, Some(900))
```

Comment:
This will want tweaking. The on press decacher is probably not aware of these new paths.